### ISSUE TITLE
pop'n peace unlock all songs doesnt work

### ISSUE THREAD

**Issue by iedesu** [mon/BemaniPatcher#127]
yes i made sure i replaced alright file, but so many songs are obviously missing which is known available on popn peace. (eg. the last world music, chaos:q) usaneko acutally worked fine btw...

**Comment by mon** [comment]
Are you 100% sure that the files are actually in the game? Since peace is n-0, perhaps your n-0 is older than what is current

**Comment by Poliwrath** [comment]
Those songs mentioned are usaneko songs and according to remywiki they're unlocked in peace as of late 2018, weird...

**Comment by iedesu** [comment]
p sure i hav correct version, checked it is 20190423, heres my dll file : https://puu.sh/FOWko/93a3a728b8.dll

**Comment by mon** [comment]
I don't have the game to compare it against, I can't really help much here

**Comment by MimiKikiNyami** [comment]
Is there a way to update more songs?

**Comment by mon** [comment]
You'll definitely have to find someone with Peace to fix the patch as I do not have it.

**Comment by MimiKikiNyami** [comment]
can i use a dll file modification program

**Comment by MimiKikiNyami** [comment]
I mean, I edit the dll to get enough songs

**Comment by mon** [comment]
Yes, you can edit the DLL to get more songs. If you work it out, post the offsets here and I can fix the patcher :)

**Comment by MimiKikiNyami** [comment]
Finally, I can't edit the dll file, sorry but I hope your team will update more songs and new ones.
